packages/web-components/src/textarea/textarea.template.ts

```diff
@@ -0,0 +1,56 @@
+import { ElementViewTemplate, html, ref, slotted } from '@microsoft/fast-element';
+import { whitespaceFilter } from '../utils/index.js';
+import type { TextArea } from './textarea.js';
+
+/**
+ * Generates a template for the TextArea component.
+ *
+ * @public
+ */
+export function textAreaTemplate<T extends TextArea>(): ElementViewTemplate<T> {
+  return html<T>`
+    <template>
+      <label ${ref('labelEl')} for="control" part="label">
+        <slot
+          name="label"
+          ${slotted({
+            property: 'labelSlottedNodes',
+            filter: whitespaceFilter,
+          })}
+        ></slot>
+      </label>
```

Comment:
I think it's being discussed but this seems like a small outlier for how we're doing labels.

Reply:
This is the same approach as `<fluent-text-input>`: https://github.com/microsoft/fluentui/blob/master/packages/web-components/src/text-input/text-input.template.ts#L20-L27.

The only difference is that the `<fluent-text-input>` uses the default slot for label, while `<fluent-textarea>` uses the `label` slot, because I have to reserve the default slot for the initial/default value for the textarea.